packages/react/src/cunningham-tokens.css

```diff
@@ -152,7 +152,7 @@
 	--c--components--forms-datepicker--border-radius--focus: 2px;
 	--c--components--forms-datepicker--border-radius--hover: 2px;
 	--c--components--forms-datepicker--border-style: solid;
-	--c--components--forms-datepicker--border-width: 2px;
+	--c--components--forms-datepicker--border-width: 1px;
```

Comment:
Have you discussed it with the designer @NathanVss? Based on the Figma design file, `border-width` should be 1px.